activesupport/lib/active_support/core_ext/numeric/conversions.rb

```diff
@@ -99,28 +99,32 @@ module ActiveSupport::NumericWithFormat
   #  1234567.to_s(:human, precision: 1,
   #                   separator: ',',
   #                   significant: false)                   # => "1,2 Million"
-  def to_s(*args)
-    format, options = args
-    options ||= {}
+  DEFAULT = Object.new
+  private_constant :DEFAULT
```

Comment:
It's private, but we're still ending up adding this to core classes. Maybe worth storing it elsewhere?